protocol/src/main/java/net/md_5/bungee/protocol/Protocol.java

```diff
@@ -321,7 +327,7 @@ public enum Protocol
                     map( ProtocolConstants.MINECRAFT_1_19_4, 0x5F ),
                     map( ProtocolConstants.MINECRAFT_1_20_2, 0x61 ),
                     map( ProtocolConstants.MINECRAFT_1_20_3, 0x63 ),
-                    map( ProtocolConstants.MINECRAFT_1_20_5, 0x65 )
```

Comment:
Is this intended?

Reply:
no i think its not thank you i will fix it